site/_docs/continuous-integration.md

```diff
@@ -118,6 +118,7 @@ customizable. If your script won't change much, you can write your test
 incantation here directly:
 
 {% highlight yaml %}
+before_install: gem install jekyll html-proofer
```

Comment:
The main article does (or should) discuss Ruby, Gemfiles, and all that great stuff.

The TL;DR / hack section provides a work around for people that don't care about all that. This PR fixes that hack. I recommend that this PR be merged on account of it being a self-standing, incremental fix.